src/com/yetanalytics/lrs_admin_ui/views/util/table.cljs

```diff
@@ -0,0 +1,42 @@
+(ns com.yetanalytics.lrs-admin-ui.views.util.table
+  (:require
+   [reagent.core :as r]
+   ["react-data-table-component" :as DataTable]))
+
+(def default-table-style
```

Comment:
Is there a reason why this CSS is inlined and not in the .sass files? I'm assuming it's because of the unique way the data table is set up...